src/lib/schematics/update/rules/attribute-selectors/attributeSelectorsStylesheetRule.ts

```diff
@@ -44,7 +44,7 @@ export class Walker extends ComponentWalker {
     this._reportExtraStylesheetFiles();
   }
 
-  visitInlineStylesheet(literal: ts.StringLiteral) {
+  visitInlineStylesheet(literal: ts.StringLiteralLike) {
```

Comment:
How did you know about this type? I tried to find something for it, but the TS compiler looks mostly undocumented.

Reply:
I most of the time just look at the source or at the `typescript.d.ts` file. The `d.ts` files are pretty good to quickly explore the public API.